lib/rack/utils.rb

```diff
@@ -7,6 +7,7 @@
 require 'tempfile'
 require 'time'
 require 'cgi/escape'
+require 'erb'
```

Comment:
You'll likely need to declare a dep on `erb` in the gemspec.

Reply:
`erb` is (for now) stdlib. Two other places here already require it.

Reply:
I do not believe that is necessary, as `erb` is stdlib and has not yet been proposed to move to bundled gems.